sky/shell/ui/engine.h

```diff
@@ -94,6 +96,8 @@ class Engine : public UIDelegate,
   gfx::Size physical_size_;
   std::string initial_route_;
   blink::SkyDisplayMetrics display_metrics_;
+  std::string language_code_;
+  std::string country_code_;
```

Comment:
Added a check in SkyView so Window (and the framework downstream) will only be notified if there has been an actual change to the language code or country code.